dev-docker

```diff
@@ -11,7 +11,7 @@ mkdir -p "$datadir"
 dockerfile="$datadir"/../Dockerfile.dev
 (
   sed -e '1,/WORKDIR/!d' "$basedir"/Dockerfile
-  echo 'RUN apt-get install -q2 snakefood flake8 git python3-wheel chromium nodejs && \'
+  echo 'RUN apt-get install -q2 snakefood flake8 git python3-wheel chromium nodejs ruby-foreman procps && \'
```

Comment:
Pretty neat, I've ran foreman a couple of times back in school,

Perhaps it would be nice to add rabbit-mq too? I've ran `./dev-docker` then `foreman start`, and the worker was never able to connect to a queue server.

Reply:
yes, good point. added rabbitmq-server as dependency and also added a rabbitmq-server process to `Procfile` (see the last commit I pushed)